pyk/src/pyk/ktool/kprove.py

```diff
@@ -24,6 +18,8 @@
 from ..prelude import mlTop
 from .kprint import KPrint
 
+_LOGGER: Final = logging.getLogger(__name__)
+
```

Comment:
This will get the same `_LOGGER` across different files?

Reply:
This will create a logger instance for the `pyk.ktool.kprove` module.

From the [docs](https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial):

> A good convention to use when naming loggers is to use a module-level logger, in each module which uses logging, named as follows:
>
> ```python3
> logger = logging.getLogger(__name__)
> ```
>
>This means that logger names track the package/module hierarchy, and it’s intuitively obvious where events are logged just from the logger name.